daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/ValueCoder.scala

```diff
@@ -185,6 +186,9 @@ object ValueCoder {
         Right(value)
     }
 
+  def decodeValue(protoValue0: proto.VersionedValue): Either[DecodeError, Value] =
+    decodeValue(CidDecoder, protoValue0)
```

Comment:
We cannot us default parameter because `decodeValue` is overloaded. 
Solution => more overloading.